attachment_category/README.rst

```diff
@@ -0,0 +1,78 @@
+==================
+Atachment Category
```

Comment:
Attachment

Reply:
@rousseldenis If you change the typo error we can proceed with the merge :wink:

Reply:
@etobella I think this is not needed as ocabot will regenerate it